bzl/bazel.rc

```diff
@@ -65,14 +65,14 @@ test:macos --test_tag_filters=-large,-linux,-msvc
 
 build:macos_x86_64_ci --cpu=darwin_x86_64
 build:macos_x86_64_ci --apple_platform_type=macos
-build:macos_x86_64_ci --build_tag_filters=-large,-linux,-msvc,-not_ci
-test:macos_x86_64_ci --test_tag_filters=-large,-linux,-msvc,-not_ci
+build:macos_x86_64_ci --build_tag_filters=-dbg,-large,-linux,-msvc,-not_ci
+test:macos_x86_64_ci --test_tag_filters=-dbg,-large,-linux,-msvc,-not_ci
 
 build:macos_x86_64_ci_dbg -c dbg
 build:macos_x86_64_ci_dbg --cpu=darwin_x86_64
 build:macos_x86_64_ci_dbg --apple_platform_type=macos
-build:macos_x86_64_ci_dbg --build_tag_filters=-dbg,-large,-linux,-msvc,-not_ci
-test:macos_x86_64_ci_dbg --test_tag_filters=-dbg,-large,-linux,-msvc,-not_ci
+build:macos_x86_64_ci_dbg --build_tag_filters=-large,-linux,-msvc,-not_ci
+test:macos_x86_64_ci_dbg --test_tag_filters=-large,-linux,-msvc,-not_ci
```

Comment:
This is backwards. I'm trying to prevent tests marked as `dbg` from running in `ci_dbg`. Perhaps it should be called `not_dbg` instead?